lua/ulx_yeet/sh_yeet.lua

```diff
@@ -186,13 +194,24 @@ local function playerDrop( ply, ent )
             if not IsValid( ragdoll ) then
                 hook.Remove( "Tick", hookName )
                 ent:Spawn()
+                ent.yeetRagdoll = nil
                 return
             end
-            if ragdoll:GetVelocity():Length() > 10 or ragdoll.cooldown > CurTime() then return end
+
+            if not IsValid( ragdoll.player ) then
+                ragdoll:Remove()
+                hook.Remove( "Tick", hookName )
+            end
+
+            if ragdoll:GetVelocity():Length() > 0.5 or ragdoll.cooldown > CurTime() then return end
```

Comment:
Two comments on this:

1. Could you move `0.5` into a variable, I don't know what it means or why it's important. (also, should it be configurable?)
2. These are, again, two separate concerns that could be split into their own return statements. We've got plenty of vertical space

Reply:
when a player reaches sub 0.5 velocity they will unragdoll, ive messed around with this value and anything higher will just feel unnatural and janky resulting in early unragdolls, i changed it because sometimes players would unragdoll when being thrown directly into the air

Reply:
Sure, sounds good. Just plop it in a variable that explains it shouldn't be changed